tests/components/homematicip_cloud/test_sensor.py

```diff
@@ -77,7 +78,7 @@
     )
 
     assert ha_state.state == "0"
-    assert ha_state.attributes[ATTR_UNIT_OF_MEASUREMENT] == "%"
+    assert ha_state.attributes[ATTR_UNIT_OF_MEASUREMENT] == UNIT_PERCENTAGE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5093397)